packages/common/src/types.ts

```diff
@@ -114,6 +114,21 @@ export interface IGetSurveyModelsResponse {
   stakeholder: IModel;
 }
 
+export interface IGetSurveyFormJson {
+  settings: {
+    [propName: string]: any,
+    openStatusInfo: {
+      status: string,
```

Comment:
can this be `status: 'scheduled' | 'closed'` or even a new enumeration like `status: SurveyStatus`

```
enum SurveyStatus {
  scheduled = "scheduled",
  closed = "closed"
}
```

Reply:
This would be fascinating to re-use [some of Survey123 types](https://github.com/ArcGIS/survey123-webform/blob/ea81d5503342bf36e06615cb0e61a13445ffd44f/libs/survey123-core/classes/form/form-settings/open-status-info.ts)

Reply:
is ^ a private repo? cannot seem to access that link

Reply:
Can't get there either...

Reply:
@ajturner I think the enum is a reasonable idea, but I would actually vote for just a custom type in this case:

`type SurveyStatus = 'open'|'scheduled'|'closed';`

That provides the type checking without requiring the dot syntax (`SurveyStatus.something`) and I think enums are most useful when we'd like to give a more descriptive name to a non-intuitive set of values.

Reply:
Also, using enums (or was it the _third_ way to skin this cat) [bit us in the ass in rest-js](https://github.com/Esri/arcgis-rest-js/pull/681). So until I understand how these things work I'm always going to use union types.